orchestrator/nodes/cron/cron.html

```diff
@@ -26,7 +26,7 @@
                 value: "",
                 validate: function(v) {
                     if($("#node-input-operation option:selected").val() === 'CREATE') {
-                        let cronregex = new RegExp(/^(\*|([0-9]|1[0-9]|2[0-9]|3[0-9]|4[0-9]|5[0-9])|\*\/([0-9]|1[0-9]|2[0-9]|3[0-9]|4[0-9]|5[0-9])) (\*|([0-9]|1[0-9]|2[0-3])|\*\/([0-9]|1[0-9]|2[0-3])) (\*|([1-9]|1[0-9]|2[0-9]|3[0-1])|\*\/([1-9]|1[0-9]|2[0-9]|3[0-1])) (\*|([1-9]|1[0-2])|\*\/([1-9]|1[0-2])) (\*|([0-6])|\*\/([0-6]))$/);
+                        let cronregex = new RegExp(/^(\*|(([0-9])|([0-5][0-9]))(((\,)(([0-9])|([0-5][0-9])))*)?(\/(([0-9])|([0-5][0-9])))?) (\*|(([0-9])|(1[0-9]|2[0-3]))(((\,)(([0-9])|(1[0-9]|2[0-3])))*)?(\/(([0-9])|(1[0-9]|2[0-3])))?) (\*|(([0-9])|(1[0-9]|2[0-9]|3[01]))(((\,)((0-9)|(1[0-9]|2[0-9]|3[01]))*))?(\/(([0-9])|(1[0-9]|2[0-9]|3[01])))?) (\*|(([0-9])|(0[0-9]|1[0-2]))(((\,)(([0-9])|(0[0-9]|1[0-2])))*)?(\/(([0-9])|(0[0-9]|1[0-2])))?) (\*|(([0-7])|(0[0-7]))((\,)(([0-7])|(0[0-7]))*)?(\/(([0-7])|(0[0-7])))?)$/);
```

Comment:
What a giant regex! :sweat_smile:

Is it used to validate Cron time string format?

You could run this one first `/(((\d+,)+\d+|(\d+(\/|-)\d+)|\d+|\*) ?){5,7}/g`, then split the string (by space) into an array of 5 or 7 positions and do a more refined validation field by field, it would be easier to read the regex in this case...

But if that is not the purpose of this regex, discard my comment.